pkgs/development/python-modules/reflex/default.nix

```
@@ -46,7 +46,7 @@
 
 buildPythonPackage rec {
   pname = "reflex";
-  version = "0.5.10";
+  version = "0.6.0";
   pyproject = true;
 
   disabled = pythonOlder "3.8";
```

Support for Python 3.8 was dropped. (https://github.com/reflex-dev/reflex/pull/3956)

```suggestion
  disabled = pythonOlder "3.10";
```

---

thanks!